tests/integrations/test_diffusion_gemma_model.py

```
@@ -0,0 +1,141 @@
+    """pixel_values / mm_token_type_ids / image_position_ids reach model.forward."""
+    from types import SimpleNamespace
+
+    from axolotl.integrations.diffusion_gemma.diffusion import DiffusionObjectiveConfig
+    from axolotl.integrations.diffusion_gemma.trainer import block_diffusion_step
+
+    seen = {}
+
+    class _RecordingModel:
+        def __call__(self, **kwargs):
+            seen.update(kwargs)
+            b, l = kwargs["decoder_input_ids"].shape
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)